interface/wx/maskededit.h

```diff
@@ -0,0 +1,935 @@
+
+        Currently there is one predefined function, wxMaskedDateShort(), which
+        uses the also predefined struct wxDateParams. Here is an example on how
+        to use it, being the date in shape yyyy/mm/dd and being the year the
+        fourth field in the control:
+        @code
+        wxDateParams dparam;
+        dparam.yearField = 3;
+        dparam.monthField = 4;
+        dparam.dayField = 5;
+        maskedCtrl->SetControlFunction( &wxMaskedDateShort, &dparam );
+        @endcode
+
+        You can also set your own function, which may use your own parameters.
+
+        @see SetFieldFunction()
+    */
+    void SetControlFunction(wxMaskedFunc* fn, void* param);
+
+    /**
+        Sets all parameters at once, replacing old ones.
+
+        @param params
+            All the parameters that govern this control: mask, flags, colours
+            and test functions.
+    */
+    bool SetParams(const wxMaskedEditParams& params);
+
+    /**
+        Sets the control's value from an undecorated text.
+
+        The plain value (i.e. undecorated text, without "literals") is supposed
+        to resemble the user's input. So this text does not need to fill all
+        cells. If this is the case, filling begins at the first field, this is,
+        from the left of the control.
+
+        @param plainValue
+            The text resembling the characters typed by the user.
+
+        If any character is not allowed, it returns @false.
+
+        If the plain value is bigger than the mask, it returns @false.
+
+        @see GetPlainValue(), SetFieldValue(), wxMaskedEditText::SetValue()
+    */
+    bool SetPlainValue(const wxString& plainValue);
+
+    /**
+        Sets the value for a field.
+
+        @param plainValue
+            The text replacing the current field's one.
+
+        If any character is not allowed, it returns @false.
+
+        @see GetFieldValue(), SetPlainValue(), wxMaskedEditText::SetValue()
+    */
+    bool SetFieldValue(size_t index, const wxString& plainValue);
+
+    /**
+        Checks whether all cells, fields and the whole control's value are OK.
+
+        Returns the position of the error found, or '-1' if everything is well.
+    */
+    long IsValid();
+
+    /**
+        Sets bell on/off when a character is not accepted.
+
+        @param bell
+            Set it to @false if you don't want the sound.
+    */
+    void SetBellOnError(bool bell);
+
+
+    //Accessors -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
+    /**
+        Gets the number of fields in the mask.
+    */
+    size_t GetFieldsCount() const;
+
+    /**
+        Gets control's value, without decorations.
+
+        The return value is the current control's value, after removing
+        all "literals" and filling untyped cells with per-field padding chars.
+
+        @see SetPlainValue(), GetFieldValue(), wxEditFieldFlags
+    */
+    wxString GetPlainValue() const;
```

Comment:
It's not clear if this is the same as the concatenation of the value of all fields. If it is, then this should probably be called `GetFieldsValue()` or something like this to show it. "Plain" is not very clear.